### PR TITLE
document: every 1.minute

### PR DESCRIPTION
**Pull request by 1c7**

### CODE REVIEW

**Comment by 1c7** [comment]
:)

File: README.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ every '0 0 27-31 * *' do
   command "echo 'you can use raw cron syntax too'"
 end
 
+every 1.minute do
```
Comment (benlangfeld) [inline README.md]: Will you also document `1.week`, `1.month`, `1.year`? Would it not be better to simply add a comment to the first example, similarly to the third, explaining that any unit of time supported by cron can be given?

Comment (1c7) [inline README.md]: you are right, let me chang this.

Comment (1c7) [inline README.md]: is this the correct way to do it ?
one more commit?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ every '0 0 27-31 * *' do
   command "echo 'you can use raw cron syntax too'"
 end
 
+
```
Comment (benlangfeld) [inline README.md]: Please remove this stray newline

File: README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ Run `whenever --help` for a complete list of options for selecting the schedule
 ### Example schedule.rb file
 
 ```ruby
-every 3.hours do
+every 3.hours do # 1.minute 1.week 1.month 1.year is also supported
```
Comment (benlangfeld) [inline README.md]: `1.day`?

Comment (1c7) [inline README.md]: sorry about that

**Pull request merged**